idsvr/templates/deployment-admin.yaml

```diff
@@ -52,8 +52,14 @@ spec:
           {{- end }}
           {{- range $env := .Values.curity.admin.extraEnv }}
             - name: {{ $env.name }}
+            {{- if $env.value }}
               value: {{ $env.value | quote }}
-          {{- end }}
+            {{- end }}
+            {{- if $env.valueFrom }}
+              valueFrom: 
+{{ $env.valueFrom | toYaml | trim | indent 16 }}
```

Comment:
looks like some formatting issues here

Reply:
you can change to `{{- $env.valueFrom | toYaml | trim | indent 16 }}` and move this to the right below `valueFrom` to be more readable. This formatting is tecnhically still correct though